src/main/java/stsjorbsmod/patches/CampfireThirstPatch.java

```diff
@@ -0,0 +1,59 @@
+package stsjorbsmod.patches;
+
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.rooms.CampfireUI;
+import com.megacrit.cardcrawl.ui.campfire.AbstractCampfireOption;
+import com.megacrit.cardcrawl.ui.campfire.RestOption;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.NewExpr;
+import stsjorbsmod.campfire.ThirstOption;
+import stsjorbsmod.characters.Cull;
+
+import java.util.Iterator;
+
+@SpirePatch(
+        clz = CampfireUI.class,
+        method = "initializeButtons"
+)
+public class CampfireThirstPatch {
+    public CampfireThirstPatch() {
+    }
+
+    public static ExprEditor Instrument() {
+       return new ExprEditor() {
+           public void edit(NewExpr e) throws CannotCompileException {
+               // Completely replace the rest option with thirst for Cull
+               if(e.getClassName().equals("com.megacrit.cardcrawl.ui.campfire.RestOption")) {
+                    e.replace(String.format(
+                            "{$_ = %1$s.getCullCampfireOption();}",
```

Comment:
It might be friendlier to other mods that want to patch the same place to do something like `"{$_ = %1$s.getCullCampfireOption($proceed($$));}"` and return the argument from `getCullCampfireOption` in the base case; that way, another mod that happens to occur after ours with an instrument/insert patch looking for the same `new RestOption` expression will still find it